### PR TITLE
UF-315: add support to IsElement for WorkbenchScreen component

### PR DESCRIPTION
**Pull request by porcelli**

### CODE REVIEW

**Comment by porcelli** [comment]
btw: the trick here is use Errai ElementWrapperWidget

File: uberfire-workbench/uberfire-workbench-processors/src/main/java/org/uberfire/annotations/processors/GeneratorUtils.java [hunk 1]
```diff
@@ -61,14 +58,15 @@
     /**
      * Handy constant for an emtpy array of argument types.
      */
-    private static final String[] NO_PARAMS = new String[0];
+    private static final String[] NO_PARAMS = new String[ 0 ];
 
     /**
      * Passing a reference to exactly this array causes
-     * {@link #getAnnotatedMethod(TypeElement, ProcessingEnvironment, String, TypeMirror, String[])} and
+     * {@link #getAnnotatedMethods(TypeElement, ProcessingEnvironment, String, TypeMirror, String[])},
```
Comment (ederign) [inline uberfire-workbench/uberfire-workbench-processors/src/main/java/org/uberfire/annotations/processors/GeneratorUtils.java]: duplicated line?

Comment (porcelli) [inline uberfire-workbench/uberfire-workbench-processors/src/main/java/org/uberfire/annotations/processors/GeneratorUtils.java]: nope, check the arg list

**Comment by porcelli** [comment]
please Jenkins re-test (error isn't related to this)

File: uberfire-workbench/uberfire-workbench-processors-tests/src/test/resources/org/uberfire/annotations/processors/expected/WorkbenchScreenTest30.expected [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+
+package org.uberfire.annotations.processors;
+
+import java.util.Arrays;
+import java.util.Collection;
+import java.util.Collections;
+import javax.annotation.Generated;
+import javax.enterprise.context.Dependent;
+import javax.inject.Inject;
+import com.google.gwt.user.client.ui.InlineLabel;
```
Comment (ederign) [inline uberfire-workbench/uberfire-workbench-processors-tests/src/test/resources/org/uberfire/annotations/processors/expected/WorkbenchScreenTest30.expected]: Please remove this.

File: uberfire-workbench/uberfire-workbench-processors-tests/src/test/resources/org/uberfire/annotations/processors/expected/WorkbenchScreenTest31.expected [hunk 3]
```diff
@@ -0,0 +1,92 @@
...
+
+package org.uberfire.annotations.processors;
+
+import java.util.Arrays;
+import java.util.Collection;
+import java.util.Collections;
+import javax.annotation.Generated;
+import javax.enterprise.context.Dependent;
+import javax.inject.Inject;
+import com.google.gwt.user.client.ui.InlineLabel;
```
Comment (ederign) [inline uberfire-workbench/uberfire-workbench-processors-tests/src/test/resources/org/uberfire/annotations/processors/expected/WorkbenchScreenTest31.expected]: also here.

File: uberfire-workbench/uberfire-workbench-processors-tests/src/test/resources/org/uberfire/annotations/processors/expected/WorkbenchScreenTest32.expected [hunk 4]
```diff
@@ -0,0 +1,92 @@
...
+
+package org.uberfire.annotations.processors;
+
+import java.util.Arrays;
+import java.util.Collection;
+import java.util.Collections;
+import javax.annotation.Generated;
+import javax.enterprise.context.Dependent;
+import javax.inject.Inject;
+import com.google.gwt.user.client.ui.InlineLabel;
```
Comment (ederign) [inline uberfire-workbench/uberfire-workbench-processors-tests/src/test/resources/org/uberfire/annotations/processors/expected/WorkbenchScreenTest32.expected]: here.

File: uberfire-workbench/uberfire-workbench-processors-tests/src/test/resources/org/uberfire/annotations/processors/expected/WorkbenchScreenTest30.expected (hunk 2 above)
Comment (porcelli) [inline uberfire-workbench/uberfire-workbench-processors-tests/src/test/resources/org/uberfire/annotations/processors/expected/WorkbenchScreenTest30.expected]: this is on template; not scope of this PR to fix it

File: uberfire-workbench/uberfire-workbench-processors-tests/src/test/resources/org/uberfire/annotations/processors/expected/WorkbenchScreenTest31.expected (hunk 3 above)
Comment (porcelli) [inline uberfire-workbench/uberfire-workbench-processors-tests/src/test/resources/org/uberfire/annotations/processors/expected/WorkbenchScreenTest31.expected]: this is on template; not scope of this PR to fix it

File: uberfire-workbench/uberfire-workbench-processors-tests/src/test/resources/org/uberfire/annotations/processors/expected/WorkbenchScreenTest32.expected (hunk 4 above)
Comment (porcelli) [inline uberfire-workbench/uberfire-workbench-processors-tests/src/test/resources/org/uberfire/annotations/processors/expected/WorkbenchScreenTest32.expected]: this is on template; not scope of this PR to fix it

**Comment by ederign** [comment]
+1

**Pull request merged**